docs/sql-ref-number-pattern.md

```diff
@@ -137,7 +137,7 @@ Note that the format string used in most of these examples expects:
 > SELECT to_number('1234-', '999999MI');
   -1234
  
--- PR indicates optional wrapping angel brakets.
+-- PR indicates optional wrapping angel brackets.
```

Comment:
Since we're here, mind taking a look and checking if there are more typos in this directory.

Reply:
> Since we're here, mind taking a look and checking if there are more typos in this directory.

I changed the other two places. Other than that, I haven't found any more obvious typos. Thank you.